src/DistributedLock.Redis/DistributedLock.Redis.csproj

```diff
@@ -49,7 +49,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="StackExchange.Redis" Version="2.2.4" />
+    <PackageReference Include="StackExchange.Redis" Version="2.6.122" />
```

Comment:
https://www.nuget.org/packages/StackExchange.Redis/2.6.122#versions-body-tab is the latest version as of today.